docs_src/tutorial_2_configuration.py

```diff
@@ -0,0 +1,40 @@
+"""
+idea https://docs.celeryproject.org/en/stable/userguide/application.html#example-3-using-a-configuration-class-object
+"""
+
+from corva import Api, Corva, Event, State
+
+app = Corva()
+
+
+class Config:
+    # 1 override default parameters for Corva instance
+
+    # 2 api params
+    api_url = 'api.localhost'
+    api_data_url = 'api.data.localhost'
+    api_key = 'api_key'
+    api_app_name = 'api_app_name'
+
+    # 3 state params
+    state_url = 'redis://'
+    state_params = {'param1': 'val1'}
+
+    # 4 other params
+    ...
+
+
+app.config_from_object(Config)
```

Comment:
we do not need a custom class for the configuration, we can pass them to the `Corva` initialization:
```
app = Corva(api_url='api.localhost', state_url='redis://', ...)
```
It would be simpler to start from this point